Main.py

```diff
@@ -1,12 +1,13 @@
 import sys
-from PySide6.QtGui import QGuiApplication
+from PySide6.QtGui import QGuiApplication, QFontDatabase
 from PySide6.QtQml import QQmlApplicationEngine
 import lib.DashboardController
 import lib.APDView
 
 app = QGuiApplication([])
 engine = QQmlApplicationEngine()
 
+QFontDatabase.addApplicationFont("fonts/Royal_Rumble_Haettenschweiler.ttf")
```

Comment:
Change already exists but is QFontDatabase.addApplicationFont("**ui**/fonts/Royal_Rumble_Haettenschweiler.ttf") instead